README.md

```diff
@@ -69,13 +69,11 @@ class MyApplication(object):
 
     def __init__(self, *args, **kwargs):
         boto = Boto(
-            logger=self.logger,
-            stats=self.stats,
+            access_key='my-access-key',
+            access_key='my-secret-key',
```

Comment:
Oh... No... It is called "typo". This is a very advanced magic trick taught only in 4th year classes.